Formula/proj.rb

```diff
@@ -1,7 +1,7 @@
 class Proj < Formula
   desc "Cartographic Projections Library"
-  homepage "https://proj4.org/"
-  url "https://download.osgeo.org/proj/proj-7.0.0.tar.gz"
+  homepage "https://proj.org/"
+  url "https://github.com/OSGeo/PROJ/releases/download/7.0.0/proj-7.0.0.tar.gz"
   sha256 "ee0e14c1bd2f9429b1a28999240304c0342ed739ebaea3d4ff44c585b1097be8"
```

Comment:
Will the `revision` need to be bumped?

Reply:
I'm not sure – it's the same `tar` file, just a new canonical download location. How do I decide whether it needs a bump? (sorry, don't do this often)

Reply:
If nothing changes in the install I don't think it should

Reply:
Great, gonna mark as resolved in that case.